DependencyInjection/Compiler/TweakCompilerPass.php

```diff
@@ -31,12 +31,31 @@ public function process(ContainerBuilder $container)
 
         $parameters = $container->getParameter('sonata_block.blocks');
 
-        foreach ($container->findTaggedServiceIds('sonata.block') as $id => $attributes) {
+        foreach ($container->findTaggedServiceIds('sonata.block') as $id => $tags) {
```

Comment:
Ok, it must be a matter of taste, so do as you wish. Be aware that `phpmd` will whine about this kind of thing though.